fhir-install/src/main/docker/ibm-fhir-schematool/DESIGN.md

```diff
@@ -14,7 +14,7 @@ The design is such that it COULD, but does not support multiple input files at t
 When the container is started, the container process one of two flows:
 
 * Schema Onboarding - creates and updates a schema and apply grants
-* Schema Offboarding - deletes a tenant (Db2) and removes data if there are no tenants left.
+* Schema Offboarding - drops a schema
```

Comment:
not a comment on this line, but a comment about this ibm-fhir-schematool design in general:   I wonder if we should remove all fancy capabilities from this image and have it be just a simple container wrapper around the cli like our other images?